README.md

```diff
@@ -150,7 +150,7 @@ Installs a given language.
 wp language core install <language>... [--activate]
 ~~~
 
-Downloads the language pack from WordPress.org.
+Downloads the language pack from WordPress.org. Find your language code at: https://translate.wordpress.org/
```

Comment:
Please note that all the content in `README.md` is derived from the source code.

You should not edit this file directly, but only update it in the source code.

Reply:
@UmeshSingla It should be updated here https://github.com/wp-cli/language-command/blob/main/src/Core_Language_Command.php#L171

Reply:
@swissspidy In core handbook, for language code, `https://make.wordpress.org/polyglots/teams/` is given in some places. Which link is more suitable here, this one or this one `https://translate.wordpress.org/` ?